experimental/add2app/SimpleIOSExample/Podfile

```diff
@@ -0,0 +1,26 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+# Added these two lines for Flutter:
+flutter_application_path = '../example_module'
+system("pushd #{flutter_application_path} && flutter create . && popd")
```

Comment:
Can you use backticks here instead?  Also, handle if flutter_application_path has spaces in it.
```
`pushd "#{flutter_application_path}" && flutter create . && popd`
```

Reply:
(In this case it doesn't have spaces, but as an example that could be pasted somewhere else, probably good to handle it).

Reply:
Great point. I've updated it.